src/xiana/migrate.clj

```diff
@@ -0,0 +1,150 @@
+(ns xiana.migrate
```

Comment:
I would keep the old namespace name `xiana.db.migrate` since this functionality has to do with database schema/data migrations. `xiana.migrate` can be misleading. one can get an impression that it is used for managing xiana version.

Reply:
I'm not very comfortable with a directory and a file with the same name (db), maybe we should keep the db folder but move db file inside with the name core, and so there would not be a single file in a directory. In my opinion, folders group files by functionality, but it still doesn't seem normal to have a directory with a single file.

Reply:
Think about it like a package. You have a `xiana.db` package, which contains a sub package `xiana.db.migrate`